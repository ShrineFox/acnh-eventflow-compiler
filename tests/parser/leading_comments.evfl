flow Test():
    # test
    # test
    return


flow Test2():

    return
