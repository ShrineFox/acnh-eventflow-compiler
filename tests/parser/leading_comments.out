RootNode[name=Test, vardefs=[], out_edges=[]]
RootNode[name=Test2, vardefs=[], out_edges=[]]

